processing/src/main/java/org/apache/druid/frame/write/FrameWriterUtils.java

```diff
@@ -251,9 +252,13 @@ public static void copyByteBufferToMemory(
       final WritableMemory dst,
       final long dstPosition,
       final int len,
-      final boolean allowNullBytes
+      final boolean allowNullBytes,
+      final boolean removeNullBytes
   )
   {
+    if (allowNullBytes && removeNullBytes) {
```

Comment:
This is quite hot piece of code. Do we wanna have this check here ?

Reply:
Instead can we make public static methods which never allow this condition to happen ?
We can make this a private method